index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Good job on the design and layout! Although, the calculator isn't fully working yet, it's heading to the right direction.